llvm/utils/release/test-release.sh

```diff
@@ -534,7 +534,7 @@ function build_llvmCore() {
       # compiler-rt builtins is needed on AIX to have a functional Phase 1 clang.
       if [ "$System" = "AIX" -o "$Phase" != "1" ]; then
         BuildTarget="$BuildTarget runtimes"
-        InstallTarget="$InstallTarget install-runtimes"
+        InstallTarget="$InstallTarget install-runtimes install-builtins"
```

Comment:
@tstellar Just wanted to check, but in the original [commit](https://github.com/llvm/llvm-project/pull/72703/commits/fcae8ce77e2256920459fb4cf428a212d6560b68) that added this section, there's no explicit `install-builtins` so did we just miss adding it or is its absence intentional?

We realized that `install-runtimes` does not actually install the compiler-rt builtins, so it appears that `install-builtins` is needed here. 

Also, it would seem that this section can run on non-AIX systems, and on Phase 2, and I'm not quite sure if that matches the comment on 534. Is this also intentional?

Reply:
> @tstellar Just wanted to check, but in the original [commit](https://github.com/llvm/llvm-project/pull/72703/commits/fcae8ce77e2256920459fb4cf428a212d6560b68) that added this section, there's no explicit `install-builtins` so did we just miss adding it or is its absence intentional?
> 


I just missed adding `install-builtins` I didn't know it was required.  Does it work if you do only `install-builtins` and not `install-runtimes`.

> 
> Also, it would seem that this section can run on non-AIX systems, and on Phase 2, and I'm not quite sure if that matches the comment on 534. Is this also intentional?

This was another mistake.  You are correct there is a bug in the current logic.

Reply:
It does work with just `install-builtins` and not `install-runtimes `

Reply:
> This was another mistake. You are correct there is a bug in the current logic.

@tstellar:
Was the intent for only AIX to have the extra targets added on Phase 1 and 2?
(I would be surprised if it was needed for only Phase 1 and not Phase 2).